cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/nukkit/listener/NukkitPlayerListener.java

```diff
@@ -41,7 +42,22 @@ public void handle(PlayerLoginEvent event) {
         String currentTaskName = Wrapper.getInstance().getServiceId().getTaskName();
         ServiceTask serviceTask = Wrapper.getInstance().getServiceTaskProvider().getServiceTask(currentTaskName);
 
-        if (serviceTask != null && serviceTask.isMaintenance() && !player.hasPermission("cloudnet.bridge.maintenance")) {
+        if (serviceTask == null) {
+            return;
+        }
+
+        // if the service has a field "requiredPermission" and the field is not null or empty and
+        // the player has the has not the permission of that field -> disconnect him
+        String requiredPermission = serviceTask.getProperties().getString("requiredPermission");
+        requiredPermission = (requiredPermission.equals("null")) ? null : requiredPermission;
+        if (requiredPermission != null &&
```

Comment:
That null check should be one line higher, it doesn't make any sense after the `requiredPermission` has been used

Reply:
yup, forgot to remove that. Pushed a fixed version since the conversion from "null" to null is not needet anymore since I'm doing that in the command if "null" is entered